tools/SetupFlow/DevHome.SetupFlow/Models/InstallPackageTask.cs

```diff
@@ -286,7 +288,11 @@ private string GetExtendedErrorCodeMessage()
 
     private void ReportAppSelectedForInstallEvent()
     {
-        TelemetryFactory.Get<ITelemetry>().Log("AppInstall_AppSelected", LogLevel.Critical, new AppInstallUserEvent(_package.Id, _package.CatalogId), _activityId);

```

Comment:
@RDMacLachlan, is it ok to rename this telemetry event name?
That is emitted when the app install begins. I found the name AppSelected to be a little inaccurate in this case.

Reply:
I will keep the original name for now, I know Roy has an email thread about this with the corresponding team. I will reconsider this change separately.